app/views/iterations/_progress_timeline.html.erb

```diff
@@ -35,8 +35,8 @@ classes = {
       <% end %>
```

Comment:
More generally might be worth considering if a partial needed for this markup.

Reply:
Turns out we don't even use this partial anymore!